docker/main/docker-build.yml

```diff
@@ -59,7 +59,17 @@ services:
       context: ./ngen
       target: rocky_ngen_build_testing
       args:
+        REPO_URL: ${NGEN_REPO_URL?No NGen repo url configured}
```

Comment:
What are your thoughts on being more explicit here and naming these `NGEN_REPO_URL`, `NGEN_BRANCH`, and `NGEN_COMMIT` instead? If I were looking at a `.env` file that contained this information and saw `REPO_URL` I would be a little confused about which repo to specify.